lib/project_types/rails/gem.rb

```diff
@@ -15,42 +15,94 @@ def install(ctx, *args)
         version = args.shift
         gem = new(ctx: ctx, name: name, version: version)
         ctx.debug(ctx.message('rails.gem.installed_debug', name, gem.installed?))
-        gem.install! unless gem.installed?
+        gem.installed? ? true : gem.install!
       end
 
       def binary_path_for(ctx, binary)
-        File.join(gem_home(ctx), 'bin', binary)
+        path_to_binary = File.join(gem_home(ctx), 'bin', binary)
+        File.exist?(path_to_binary) ? path_to_binary : binary
       end
 
       def gem_home(ctx)
         ctx.getenv('GEM_HOME') || apply_gem_home(ctx)
       end
 
+      def gem_path(ctx)
```

Comment:
same comment as before, this might need a better name since it is both getting and setting the GEM_PATH